src/main/kotlin/io/bazel/kotlin/plugin/jdeps/JdepsGenExtension.kt

```diff
@@ -265,7 +265,10 @@ class JdepsGenExtension(
     val jarsToClasses = mutableMapOf<String, MutableList<String>>()
     classes.forEach {
       val parts = it.split("!/")
-      jarsToClasses.computeIfAbsent(parts[0]) { ArrayList() }.add(parts[1])
+      val jarPath = parts[0]
+      if (jarPath.endsWith(".jar")) {
+        jarsToClasses.computeIfAbsent(jarPath) { ArrayList() }.add(parts[1])
```

Comment:
The logic that relies on this is highly reliant on jar stamping to correctly work, which isn't always enabled by default. Not sure what the best way to handle this is.

Reply:
Is the concern on third party jars being unstamped? We could force it in the rules...

Reply:
Should have been more specific here. This comment was more of a mental note-to-self as we work on jdeps/classpath reduction.